unfollow/unfollow.py

```diff
@@ -124,17 +125,16 @@ def get_unfollows(username):
 
     # get newly added 'current' followers and load to file
     with open(f"{UNFOLLOW_PATH}/followers.json") as follower_file:
-       for follower in follower_file:
-           current_followers.append(follower)
+        current_followers.extend(iter(follower_file))
 
     # compare
     for old_follower in previous_followers:
         result = scan_follows(old_follower, current_followers)
         result_for_link = result
-        if result != True:
+        if result is not True:
             if check_deleted(result):
-                result=f"{result} \[deleted]"
-            unfollowers.append((result,f"https://github.com/{result_for_link}/"))
+                result = f"{result}[deleted]"
```

Comment:
Please keep `result=f"{result} \[deleted]"` as it was previously! `[deleted]` is actual text and an escape character is required so python doesn't confuse it with the `unfollowers` list.

Of course adding a space between the equals sign is fine tho 👍 `result = f"{result} \[deleted]"`